backend/rates/rates.go

```diff
@@ -119,15 +120,15 @@ type RateUpdater struct {
 	// See https://www.coingecko.com/en/api for details.
 	coingeckoURL string
 	// All requests to coingeckoURL are rate-limited using geckoLimiter.
-	geckoLimiter *ratelimit.LimitedCall
+	geckoLimiter *rate.Limiter
 }
 
 // NewRateUpdater returns a new rates updater.
 // The dbdir argument is the location of a historical rates database cache.
 // The returned updater can function without a valid database cache but may be
 // impacted by rate limits. The database cache is transparent to the updater users.
 // To stay within acceptable rate limits defined by CoinGeckoRateLimit, callers can
-// use util/ratelimit package.
+// use https://pkg.go.dev/golang.org/x/time/rate package.
```

Comment:
remove this part? users don't have to do anything, rate limiting is includd in the RatesUpdater